common/remote/sockfile.cpp

```diff
@@ -1269,8 +1269,10 @@ protected: friend class CRemoteFileIO;
                         msg.appendf(" %2x",(int)rest[i]);
                 }
             }
-            else if(errCode == 8209)
+            else if (errCode == 8209)
                 msg.append("Failed to open directory.");
+            else if (errCode == RFSERR_GetDirFailed)
```

Comment:
Lines 1272 and 1273 should be deleted, otherwise this line can't be reached.

Reply:
Why? The RFSERR_GetDirFailed==8019 (as you can see in remoteerr.hpp) and not 8209. There are some multiple local (re)definition for RFSERR_GetDirFailed macro. I think I will open a JIRA to make one global definition for it.

I left that check for compatibility consideration (if some dafilesrv uses 8209 value instead of 8019.) 
Perhaps I should add the 8209 check to the size as well.

Reply:
hm, multiple different definitions of RFSERR_GetDirFailed, yuk - yes I can see them now.

in rfs.cpp
```
#define RFSERR_GetDirFailed                     8019
```

and in daliservix.cpp
```
#define RFSERR_GetDirFailed                     8209
```

The daliservix is supposed to be a solaris version of dafilesrv.
Why they have different codes... I do not know, but best leave it as is then.